phpBB/phpbb/avatar/driver/gravatar.php

```diff
@@ -180,6 +182,17 @@ protected function get_gravatar_url($row)
 			$url .= '?s=' . max($row['avatar_width'], $row['avatar_height']);
 		}
 
+		/**
+		* Modify gravatar url
+		*
+		* @event core.get_gravatar_url_after
+		* @var	string	row	User data or group data
+		* @var	string	url	Gravatar URL
+		* @since 3.1.7-RC1
+		*/
+		$vars = array('row', 'url');
+		extract($phpbb_dispatcher->trigger_event('core.get_gravatar_url_after', compact($vars)));
```

Comment:
I'm failing to see why we would need to have an event here, could you give me a use case please?